CHANGELOG.md

```diff
@@ -2,7 +2,17 @@
 # Changelog
 All notable changes to this project will be documented in this file. 
 
-# [3.0.1] - 2022-02-21
+# [3.0.3] - UNREALEASED
+
+- BFV: the plaintext modulus `T` can now be a factor of `Q`. If so, it is always assumed to be the first moduli.
+- BFV: partially introduced the concept of `level`, a ciphertext can be rescaled with `QuantizeToLvl` to erase its lowest bits, which can be used to enforce circuit privacy. Rescaled ciphertexts can still be operated on, but operations must be done between ciphertexts of the same level to ensure correctness.
```

Comment:
Its partial because operations between ciphertexts/plaintexts of different levels are not supported. The last sentence is "true", but in the implementation of the CKKS scheme this is managed automatically, but it is not the case for BFV.

Reply:
TODO: 

- [ ] Encoding/Plaintext instantiation at level
- [ ] Evaluator checks matching input levels 
- [ ] Evaluator takes input level into account for polynomial evaluation.

Reply:
Done